sdk/cdn/arm-cdn/src/operations/customDomains.ts

```diff
@@ -200,16 +201,40 @@ export class CustomDomains {
    */
   enableCustomHttps(resourceGroupName: string, profileName: string, endpointName: string, customDomainName: string, options: Models.CustomDomainsEnableCustomHttpsOptionalParams, callback: msRest.ServiceCallback<Models.CustomDomain>): void;
   enableCustomHttps(resourceGroupName: string, profileName: string, endpointName: string, customDomainName: string, options?: Models.CustomDomainsEnableCustomHttpsOptionalParams | msRest.ServiceCallback<Models.CustomDomain>, callback?: msRest.ServiceCallback<Models.CustomDomain>): Promise<Models.CustomDomainsEnableCustomHttpsResponse> {
-    return this.client.sendOperationRequest(
-      {
-        resourceGroupName,
-        profileName,
-        endpointName,
-        customDomainName,
-        options
-      },
-      enableCustomHttpsOperationSpec,
-      callback) as Promise<Models.CustomDomainsEnableCustomHttpsResponse>;
+    let newOptions: Models.CustomDomainsEnableCustomHttpsOptionalParams = {};
```

Comment:
The next time we auto generate this package, the hand written code here will get over written. Please use the `#region` notation to mark this block of code as hand written. This way when we review the next auto generated code and see this block deleted, we can add it back knowing that this was added intentionally.